src/components/UserMasterPanel.js

```diff
@@ -309,6 +328,9 @@ const UserMasterPanel = (props) => {
             </InputAdornment>
           }
         />
+        <Typography color={passwordScore >= 2 ? "primary" : "error"} className={classes.passwordFeedback}>
```

Comment:
I suggest you to create a variable which will explicitly say what do you check here. So instead of `color={passwordScore >= 2 ? "primary" : "error"}` I would declare a variable: `const IS_PASSWORD_SECURED = passwordScore >= 2;` and then use it like that: `color={IS_PASSWORD_SECURED ? "primary" : "error"}`. It will be more readable.